### DISCUSSION TITLE
Appreciation Thread

### DISCUSSION THREAD

**Discussion by carlosssanchez** [microfeed/microfeed#101] [General]
I found microfeed yesterday randomly across the web and set it up in no time without any issues! I'm truly impressed by everything you were able to pack into it and it is exactly what I was looking for. Lately I haven't been able to say that about most software projects I start, especially blogging platforms...so I wanted to leave this shout out here to let you know how much I/we appreciate it!

**Comment by wenbinf** [comment]
Thanks for your kind words :)
